src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -574,7 +639,8 @@ public override async Task InvokeMethodGrpcAsync(string appId, string methodName
 
         var envelope = new Autogenerated.InvokeServiceRequest()
         {
-            Id = appId, Message = new Autogenerated.InvokeRequest() { Method = methodName, },
+            Id = appId,
```

Comment:
To this and the remaining formatting changes - same as above, please don't submit these generally.